scripts/stairwell_install_win.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+    Use this script to install the Stairwell forwarder on Windows OS machines.
+    This script requires your file forwarder token and environment id.
+    Visit https://docs.stairwell.com for further assistance.
+
+    Typically this would be used for remotely installing, such as in an Azure scale set
+    ex. Set-AzVMCustomScriptExtension -ResourceGroupName "ResourceGroup11" -Location "Central US" -VMName "VirtualMachine07" -Name "StairwellScriptExt" -FileName "stairwell_install_win.ps1" -Argument 'ForwarderToken EnvironmentId' | Update-AzureVM
+#>
+param(
+    [Parameter(Mandatory=$True)]
+    [string]$ForwarderToken,
+
+    [Parameter(Mandatory=$True)]
+    [string]$EnvironmentId
+)
+
+# Check for C:\Temp directory
+if (!(Test-Path "C:\Temp")) {
```

Comment:
I'd be tempted to use `$env:temp`

Reply:
I would definitely recommend using the temp environment variable. Chances are, c:\temp exists, and is writable, but using the user profile's assigned temp directory is more correct.